image_client/picturae_importer.py

```diff
@@ -56,7 +56,7 @@ def init_all_vars(self, date_string, paths):
         # full collector list is for populating existing and missing agents into collector table
         # new_collector_list is only for adding new agents to agent table.
         empty_lists = ['barcode_list', 'image_list', 'full_collector_list', 'new_collector_list',
-                       'taxon_list', 'new_taxa']
+                       'taxon_list', 'new_taxa', 'parent_list']
```

Comment:
added parent list class level parameter, to allow use of parent list between multiple methods.